src/actions/loading.js

```diff
@@ -0,0 +1,14 @@
+import * as actionTypes from "./types"
+
+export const startLoading = (message = "") => dispatch => {
+  dispatch({
```

Comment:
You don't have to use dispatch if your action only dispatch something

Reply:
Nice